kratos/utilities/os_utilities.cpp

```diff
@@ -0,0 +1,45 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+// System includes
+#include <stdio.h>  /* defines FILENAME_MAX */
+#ifdef _WIN32
```

Comment:
There is a macro for this at the top of define.h

Reply:
Done, I dind't know it